.pylintrc

```diff
@@ -0,0 +1,551 @@
+[MASTER]
```

Comment:
Where did you get this file?

Reply:
This is generated by pylint. This is the default, so we can tweak it in the future.

Reply:
... and share between projects.